README.md

```diff
@@ -74,13 +79,26 @@ wget https://github.com/crystaldb/autodba/releases/download/v0.1.0/install.sh
 
 PostgreSQL connection strings (`<CONNECTION_STRING_TO_YOUR_POSTGRES_DB>`) should be of the form `postgres://<POSTGRES_USERNAME>:<POSTGRES_PASSWORD>@<HOST>:<PORT>/<DATABASE_NAME>`.
 
-3. Run the installation script
+4. Run the make command to install AutoDBA.
+
+For system-wide installation:
+
+```bash
+sudo make install config=autodba_config.json system=1
+```
+
+Or for a user-specific installation, specify your preferred install directory:
+
+```bash
+make install config=autodba_config.json install_dir=$HOME/autodba
+```
 
+Or to install in the same extracted directory:
 ```bash
-sudo bash install.sh --system --package autodba-0.1.0.tar.gz --config autodba_config.json
+make install config=autodba_config.json
```

Comment:
I got the following error.
`make: *** No rule to make target `install'.  Stop.`